aikau/src/main/resources/alfresco/services/DocumentService.js

```diff
@@ -329,7 +325,12 @@ define(["dojo/_base/declare",
          }
          else
          {
-            url = AlfConstants.URL_SERVICECONTEXT + "components/documentlibrary/data/doclist/" + params;
+            url = AlfConstants.URL_SERVICECONTEXT + "components/documentlibrary/data/";
+            if (payload.site)
```

Comment:
OK... I hadn't realised that. Well, it was working in Share, so I guess that makes sense... in which case, it will be necessary to push some changes to the mocking service (aikau/src/test/resources/testApp/js/aikau/testing/mockservices/FullDocLibMockXhr.j).

In the `setupServer` function the first call to `this.server.respondWith` needs to be updated.

So...

```JavaScript
/\/aikau\/service\/components\/documentlibrary\/data\/doclist\/all\/site\/site1\/documentlibrary\?filter=path(.*)/,
```

...needs to become...

```JavaScript
/\/aikau\/service\/components\/documentlibrary\/data\/site\/site1\/doclist\/all\/site\/site1\/documentlibrary\?filter=path(.*)/,
```